api/v0/observe/observe.proto

```diff
@@ -32,6 +32,8 @@ syntax = "proto3";
 
 package aurae.observe.v0;
 
+option go_package = "github.com/aurae-runtime/ae/client/pkg/api/observe/v0;observev0";
```

Comment:
We might want to be using a Go structure of `pkg/api/v0/observe` instead of `pkg/api/observe/v0` to match the proto structure. `observer|runtime|schedule` are all scoped under the same version.

Reply:
Yes, it does make sense

Reply:
OH yes we sure mirror the same structure please